_includes/about.html

```diff
@@ -0,0 +1,33 @@
+<section class="about-and-video">
+  <div class="container about">
+    <div class="info">
+      <h3>About Ember &#123;&#123;fest&#125;&#125;</h3>
+      <div class="text-and-button">
+        <p>
+          EmberFest is the European Community Ember Conference. If you’re
+          looking for updates on the latest and greatest in Ember and Glimmer
+          this is the place to be. EmberFest is also a great opportunity to get
+          in touch with the European Ember Community (and friends from abroad)
+          and hiring Ember talent.
+        </p>
+        <div class="button get-involved">
+          <a alt="get involved!" href="#">Get involved!</a>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="container video-2022">
+    <h4>Ember &#123;&#123;fest&#125;&#125; 2022</h4>
+    <iframe
+      src="https://www.youtube.com/embed/irL80bQ8dHE"
+      frameborder="0"
+      allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+      allowfullscreen
+    ></iframe>
+    <p>
+      While we're getting ready for EmberFest 2023 in Madrid, check out the
```

Comment:
```suggestion
      While we're getting ready for EmberFest 2023, check out the
```

We're not announcing the host city yet